nuxt.config.ts

```diff
@@ -56,6 +56,7 @@ export default defineNuxtConfig({
       pdfViewerOrigin: process.env.PDF_VIEWER_ORIGIN,
       pdfViewerURL: `${process.env.PDF_VIEWER_ORIGIN}${process.env.PDF_VIEWER_PATH}`,
       sentryDsn: process.env.SENTRY_DSN,
+      v3PublicURL: process.env.V3_PUBLIC_BASE_URL,
```

Comment:
Please sort the key alphabetically
```suggestion
      baseURL: process.env.BASE_URL,
```

Reply:
[🔧 Rename V3_PUBLIC_BASE_URL to BASE_URL](https://github.com/likecoin/liker-land-v3/pull/42/commits/1a3844a9daa9f3fcb7295aef9dbc7b2dbbba62e5)